.knip.jsonc

```diff
@@ -0,0 +1,37 @@
+{
+  "entry": [
+    "src/main.tsx",
+    "src/scrivitoExtensions.tsx",
+    "src/dumpContent.ts",
+    "functions/auth/*.ts",
+
+    "src/Data/**/*DataClass.ts",
+    "src/Data/**/*DataItem.ts",
+    "src/Data/**/*EditingConfig.ts",
+    "src/Data/**/*EditingConfig.tsx",
+    "src/Data/**/*ValuesConfig.ts",
+
+    "src/Objs/**/*Component.tsx",
+    "src/Objs/**/*EditingConfig.ts",
+    "src/Objs/**/*EditingConfig.tsx",
+    "src/Objs/**/*LayoutComponent.tsx",
+    "src/Objs/**/*ObjClass.ts",
+
+    "src/Widgets/**/*Component.tsx",
+    "src/Widgets/**/*EditingConfig.ts",
+    "src/Widgets/**/*EditingConfig.tsx",
+    "src/Widgets/**/*LayoutComponent.tsx",
+    "src/Widgets/**/*WidgetClass.ts",
```

Comment:
I wasn't able to make `import.meta.glob` work with knip. That is why I whitelisted all those files here as well.

Reply:
I found https://github.com/webpro-nl/knip/issues/168, but it looks like a custom plugin is needed.